Sources/Prometheus/PrometheusCollectorRegistry.swift

```diff
@@ -52,22 +52,38 @@ public final class PrometheusCollectorRegistry: Sendable {
         let help: String
     }
 
+    private enum BucketType: Sendable, Hashable {
```

Comment:
Doesn't feel like a "type" it is the actual buckets - plural because of collection of bucket which is one value in here

```suggestion
    private enum HistogramBuckets: Sendable, Hashable {
```